web/src/assets/locales/en.json

```diff
@@ -255,6 +255,7 @@
         "log_auto_scroll_off": "Turn off automatic scrolling"
       },
       "protected": {
+        "review": "Review changes",
         "awaits": "This pipeline is awaiting approval by some maintainer!",
```

Comment:
Can we change this also:
```suggestion
        "awaits": "This pipeline is awaiting approval by a maintainer!",
```

Reply:
Unrelated to this PR but I can add it.

Reply:
Yes I know, but that sentence always seemed kinda strange :)